### DISCUSSION TITLE
how enable cors

### DISCUSSION THREAD

**Discussion by borodadada** [nuxt/nuxt#16214] [Questions] [answered]
Hi! How enable cors? my http requset not working.
```
Access to fetch at 'http://ipwho.is/' from origin 'http://localhost:3011' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```
How fix it?

**Comment by TheAlexLichter** [comment, marked as answer]
That is up to the site you request.
What you can do is to proxy the call, e.g. through a `/server` api endpoint.

**Reply by borodadada** [reply]
i need get IP info from browser (without server)
i try
```
onMounted(async() => {
    myIp.value = await get_ip()
});
```
```
const ip = await $fetch('http://ip-api.com/json/', { method: 'GET' })
```
browser block this call

**Reply by IsraelOrtuno** [reply]
Check this https://github.com/ipapi-co/api/issues/2

Also, as @manniL suggested creating a proxy using https://nuxt.com/docs/guide/directory-structure/server#server-directory would help to bypass the CORS issues

**Reply by borodadada** [reply]
thx man all fine!
